src/Sulu/Bundle/CategoryBundle/Controller/CategoryController.php

```diff
@@ -194,11 +164,11 @@ public function cgetAction(Request $request)
      */
     public function postAction(Request $request)
     {
-        return $this->saveEntity($request, null);
+        return $this->saveCategory($request);
     }
 
     /**
-     * Changes an existing category.
+     * Replaces the category which is assigned to the given id.
```

Comment:
@danrot: depends on the definition of replace/update. 
the put method sets all category-properties which are accesible by a get request. if you send a put request which only contains the id and the name, other accessible properties (like key) are deleted. in my opinion, an update would update the name but not remove the key (this is what an patch-request does).